src/requirements/data/majors/psych.ts

```diff
@@ -0,0 +1,202 @@
+import { Course, CollegeOrMajorRequirement } from '../../types';
+import {
+  includesWithSubRequirements,
+  includesWithSingleRequirement,
+  courseMatchesCodeOptions,
+  ifCodeMatch,
+} from '../checkers-common';
+
+const psychRequirements: readonly CollegeOrMajorRequirement[] = [
+{
+    name: 'Credit Requirements',
+    description:
+        'A total of 40 credits in psychology (including prerequisites), from which students majoring in psychology are expected to choose, in consultation with their advisors, a range of courses that covers the basic processes in psychology.',
+    source:
+        'https://psychology.cornell.edu/major',
+    checker: includesWithSingleRequirement(
```

Comment:
don't we have a function to include all PSYCH prefix courses? that'd be cleaner and more future-proof than listing very PSYCH course

Reply:
I agree! We don't have a function (yet) but it should be easy to refactor. For now we can use wildcards: `includesWithSingleRequirement('PSYCH ****')`.